### PR TITLE
When adding IPs and this is the first location, hide locations dropdown (+minor styling fix)

### PR DESCRIPTION
**Pull request by kjdchapman**
I don't think this needs any new tests, but I'm happy to hear opinions.

(Cheeky to put aligning in the same PR, can rebase rather than squash to keep them separate)

The form without locations, and an example of the right-aligned button looking nice:

image

image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -17,3 +17,7 @@ $govuk-font-url-function: "font-url";
 .govuk-phase-banner{
   margin-bottom: 25px;
 }
+
+.text-right{
+  text-align: right;
+}
```
Comment (emileswarts) [inline app/assets/stylesheets/application.scss]: Definitely sure nothing in the frontend styling we import that gives us this?

File: app/views/ips/new.html.erb [hunk 2]
```diff
@@ -9,12 +9,14 @@
...
-      </div>
+      <% if @locations.present? %>
+        <div class="govuk-form-group <%= field_error(@ip, :location_id) %>">
+          <%= form.label :location_id, "Add an IP to an existing location", class: "govuk-label" %>
+          <%= form.select :location_id, @locations, { prompt: '-- select --' }, { class: "govuk-select" } %>
+        </div>
 
-      <p class="govuk-body"><strong>OR</strong></p>
+        <p class="govuk-body"><strong>OR</strong></p>
+      <% end %>
```
Comment (emileswarts) [inline app/views/ips/new.html.erb]: Nice fix 👍

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (kjdchapman) [reply app/assets/stylesheets/application.scss]: About 80% - had a look at some of the other admin tools and this seems to be what they were doing. Had a quick look through the Design System too, no mention of it.

Comment (emileswarts) [reply app/assets/stylesheets/application.scss]: Okay I see

**Pull request merged**